### PR TITLE
Introduce method that returns raw logits for individual estimators

### PR DESCRIPTION
**Pull request by bejaeger**
## Motivation and Context

Provide a way to write out logits of individual estimators to perform analyses on the logit level afterwards.

---

## Public API Changes

-   [X] No Public API changes
-   [ ] Yes, Public API changes (Details below)

---

## How Has This Been Tested?

---

## Checklist

-   [X] The changes have been tested locally.
-   [X] Documentation has been updated (if the public API or usage changes).
-   [ ] A entry has been added to `CHANGELOG.md` (if relevant for users).
-   [X] The code follows the project's style guidelines.
-   [X] I have considered the impact of these changes on the public API.

---

### CODE REVIEW

**Review by oscarkey — APPROVED** [review]

**Comment by carterprince** [comment]
hi -- are there plans to add this to `regressor.py` as well?
